### PR TITLE
Qc state model refactoring

### PR DESCRIPTION
**Pull request by mgcam**

### CODE REVIEW

File: lang_qc/endpoints/pacbio_well.py [hunk 1]
```diff
@@ -28,7 +28,8 @@
 
 from lang_qc.db.mlwh_connection import get_mlwh_db
 from lang_qc.db.qc_connection import get_qc_db
-from lang_qc.db.qc_schema import QcState, QcStateHist
+from lang_qc.db.qc_schema import QcState as QcStateDB
```
Comment (nerdstrike) [inline lang_qc/endpoints/pacbio_well.py:31]: FYI, it also acceptable to do `from lang_qc.db.qc_schema import QcState as QcStateDB, QcStateHist as QcStateDBHist`, perhaps a bit too long for this case

File: lang_qc/models/qc_state.py [hunk 2]
```diff
@@ -18,24 +18,88 @@
...
+        attribute of this object was assigned.
+        """,
+    )
+    user: str = Field(default=None, title="User who assigned current QC state")
+    created_by: str = Field(
+        default=None,
+        title="This QC state assignment context",
+        description="""
+        QC state assignment context, i.e. the name of the web application,
+        script name, JIRA ticket number, etc., which i sassociated with the
```
Comment (nerdstrike) [inline lang_qc/models/qc_state.py:76]: Spacing issue/typo

File: lang_qc/util/qc_state_helpers.py [hunk 3]
```diff
@@ -24,17 +24,17 @@
 from sqlalchemy import and_, select
 from sqlalchemy.orm import Session
 
+from lang_qc.db.qc_schema import ProductLayout
+from lang_qc.db.qc_schema import QcState as QcState
```
Comment (nerdstrike) [inline lang_qc/util/qc_state_helpers.py:28]: Alias to itself is a little eccentric.

**Review by nerdstrike — CHANGES_REQUESTED** [review]

File: lang_qc/endpoints/pacbio_well.py (hunk 1 above)
Comment (mgcam) [reply lang_qc/endpoints/pacbio_well.py:31]: This is enforced by isort

File: lang_qc/util/qc_state_helpers.py (hunk 3 above)
Comment (mgcam) [reply lang_qc/util/qc_state_helpers.py:28]: This is enforced by isort

File: lang_qc/models/qc_state.py (hunk 2 above)
Comment (mgcam) [reply lang_qc/models/qc_state.py]: Thank you, @nerdstrike, fixed.

**Review by nerdstrike — APPROVED** [review]

**Pull request merged**